Permissions/Permissions/Private/Main.cpp

```diff
@@ -925,13 +915,73 @@ namespace Permissions
 		file.close();
 	}
 
+	void ReloadConfig(APlayerController* player_controller, FString* /*unused*/, bool /*unused*/)
+	{
+		auto* shooter_controller = static_cast<AShooterPlayerController*>(player_controller);
+
+		try
+		{
+			ReadConfig();
+
+			HideAllPlayerSuccessMessages = config.value("HideAllPlayerSuccessMessages", false);
+			SendMessagesAsNotification = config.value("SendMessagesAsNotification", false);
+
+			TextSize = config.value("TextSize", 1.5f);
+			DisplayTime = config.value("DisplayTime", 3.0f);
+
+		}
+		catch (const std::exception& error)
+		{
+			AsaApi::GetApiUtils().SendServerMessage(shooter_controller, FColorList::Red, "Failed to reload config");
+
+			Log::GetLog()->error(error.what());
+			return;
+		}
+
+		AsaApi::GetApiUtils().SendServerMessage(shooter_controller, FColorList::Green, "Reloaded config");
+	}
+
+	void ReloadConfigRcon(RCONClientConnection* rcon_connection, RCONPacket* rcon_packet, UWorld* /*unused*/)
+	{
+		FString reply;
+
+		try
+		{
+			ReadConfig();
+
+			HideAllPlayerSuccessMessages = config.value("HideAllPlayerSuccessMessages", false);
+			SendMessagesAsNotification = config.value("SendMessagesAsNotification", false);
```

Comment:
These 4 new config option lines should be in `ReadConfig()` if you are loading global variables.

Reply:
All 3 times this code is repeated needs to be moved to `ReadConfig()`